src/urllib3/util/retry.py

```diff
@@ -159,13 +208,16 @@ class Retry(object):
         request.
     """
 
-    DEFAULT_METHOD_WHITELIST = frozenset(
+    #: Default methods to be used for ``method_allowlist``
+    DEFAULT_METHOD_ALLOWLIST = frozenset(
```

Comment:
0.5 vote for `DEFAULT_ALLOWED_METHODS` if we're not going for symmetry with the remove headers variable naming. (Does it just add confusion that it's not actually a list?)

Reply:
That's true, I'll make this change.